### ISSUE TITLE
Update repo to mark V spec as ratified

### ISSUE THREAD

**Issue by hdelassus** [riscv/riscv-v-spec#860]
Related issue: https://github.com/riscv/riscv-isa-manual/issues/981

It looks like V spec was ratified in 2021, right? This repo could have its description updated, as well as its README, and probably other text mentioning "Frozen" :)

**Comment by appujee** [comment]
Does the recent update in: https://github.com/riscv/riscv-v-spec/commit/e2ccd0548d6ce0b769ecdd1ce2521c19e26c39de address this issue?

**Comment by sorear** [comment]
The first paragraph of README.md still claims that ratification will not occur until V reaches 2.0.

**Comment by hdelassus** [comment]
Good point. I think it's the other way around, though: "Once ratified, the spec will be given version 2.0."
